src/lucky/redirectable.cr

```diff
@@ -42,22 +42,23 @@ module Lucky::Redirectable
     redirect to: action.route, status: status
   end
 
-  # Redirect using a `String` and a `Status` value
+  # Redirect to the given path, with a human friendly status
   #
   # ```crystal
   # redirect to: "/users", status: Status::MovedPermanently
```

Comment:
Oh and can you add an example using a symbol with a comment saying Crystal will convert it to the matching enum at compile time?